### ISSUE TITLE
Document that Ganeti doesn't support DRBD second replica

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#307]
Originally reported of Google Code with ID 251.

```
Hello,

gnt-cluster version
Software version: 2.5.0
Internode protocol: 2050000
Configuration format: 2050000
OS api version: 20
Export interface: 0


drbd version : 8.3.11

DRBD 8.3  supports a second replica. Does ganeti support it also? I did not find in ganeti docs about second replica.

thanks,
Tigran
```
Originally added on 2012-07-12 11:23:20 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

DRBD 8.3 doesn't support a second replica, it supports a "stacked" setup where you duplicate/stack two DRBD devices on top of each other.

As such, this setup is not working with live migration, since it's not a fully-symmetric setup, and Ganeti doesn't support it. We should update the documentation to mention this, thanks for the reminder.
```
Originally added on 2012-07-19 08:50:41 +0000 UTC.

Added Labels: Component-docs
Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-07-19 09:26:55 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as fixed, this has been implemented in commit 24ccfd7.
```
Originally added on 2012-07-19 15:52:07 +0000 UTC.

Added Labels: Type-Documentation
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 09:14:55 +0000 UTC.

Changed State: Released
